app/views/posts/_awards_modal.html.slim

```diff
@@ -0,0 +1,15 @@
+div#awardsSelectModal.modal.fade tabindex="-1" aria-labelledby="awardsSelectModal" aria-hidden="true"
+  .modal-dialog.modal-dialog-centered.modal-dialog-scrollable
+    .modal-content
+      .modal-header
+        h5.modal-title
+          | Choose reward
+      .modal-body
+        .row
+          - Award.all.includes(:image_blob).each do |award|
+            label.col-md-4
+              input type="radio" name="award" value="#{award.id}"
+              = image_tag award.image, style: "width:64px;height:64px;", class: "js-award", title: award.title, data: { award: award.id, post: @post.id, toggle: "tootlip" }
```

Comment:
Man ir aizdomas kā par to jau rakstiju, tev vajag style uz css parnest.
Šoreiz viss ok būs, jā tev būs divas klases class: "award js-award"

Tāpēc kā viena klase būs priekš CSS un otra prieksš Javascript

Reply:
galigi nepamaniju to style, likas ka jau bija iznemts 😄